src/views/About/about.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../vars';
+
+p{
```

Comment:
We don't have a Sass linter set up (maybe we should!), but it is conventional to put a space between a selector (`p` in this situation) and the opening curly brace (`{`).